tests/framework/base/BaseObjectTest.php

```diff
@@ -64,25 +65,31 @@ public function testCanSetProperty()
         $this->assertFalse($this->object->canSetProperty('Content'));
     }
 
+    /**
+     * @expectedException \yii\base\UnknownPropertyException
+     */
```

Comment:
Do not use annotations for expected exception: https://thephp.cc/news/2016/02/questioning-phpunit-best-practices

Reply:
OK, fixed. I suggest to add this to contribution guide